core/domain/beam_job_services_test.py

```diff
@@ -178,6 +182,34 @@ def test_get_beam_job_runs_with_refresh(self):
         self.assert_domains_equal_models(
             beam_job_runs, updated_beam_job_run_models)
 
+    def test_get_beam_job_runs_with_refresh_of_non_dataflow_job(self):
```

Comment:
It took me a while to understand what "non dataflow job" meant. Maybe something like "in dev mode" instead, and a comment explaining that the behaviour on the local server differs from that on the prod server and why?

Ditto below.

Reply:
Done.